app/models/grda_warehouse/utility.rb

```diff
@@ -151,6 +151,7 @@ def self.modifier(model)
     cascade_models = [
       GrdaWarehouse::DataSource,
       GrdaWarehouse::Hud::Client,
+      GrdaWarehouse::Hud::Project,
       GrdaWarehouse::ServiceHistoryEnrollment,
     ]
     return 'CASCADE' if cascade_models.include?(model)
```

Comment:
That should be fine, this is only used in tests and occasionally in dev.  I believe it will require some data that cascades to correctly work though.  (My memory on that is not clear.)